app/assets/scripts/views/analytics-index.js

```diff
@@ -10,30 +12,52 @@ var AnalyticsIndex = React.createClass({
 
   propTypes: {
     children: React.PropTypes.object,
+    _fetchProvinces: React.PropTypes.func,
     _fetchVProMMsids: React.PropTypes.func,
+    _fetchVProMMsidsProperties: React.PropTypes.func,
+    _fetchFieldVProMMsids: React.PropTypes.func,
+    _setCrossWalk: React.PropTypes.func,
+    provincesFetched: React.PropTypes.bool,
+    VProMMsFetched: React.PropTypes.bool,
+    provinces: React.PropTypes.array,
+    crosswalk: React.PropTypes.object,
+    params: React.PropTypes.object,
     VProMMSids: React.PropTypes.object,
-    params: React.PropTypes.object
+    VProMMs: React.PropTypes.object,
+    fieldIds: React.PropTypes.array,
+    fieldIdsFetched: React.PropTypes.bool
   },
 
-  render: function () {
-    let accumulator = { done: 0, total: 0 };
-    const provinceData = _.map(this.props.VProMMSids.data, (data, id) => {
-      const name = data.provinceName;
-      const done = data.vpromms.filter(v => v.inTheDatabase).length;
-      const total = data.vpromms.length;
-      accumulator.done += done;
-      accumulator.total += total;
-      const percentageComplete = (done / total).toFixed(2);
+  componentWillMount: function () {
+    this.props._fetchProvinces();
+    this.props._fetchFieldVProMMsids();
+    this.props._fetchVProMMsidsProperties();
+  },
+
+  renderAnalyticsIndex: function () {
+    const vprommsProvinceKeys = Object.keys(_.invert(this.props.crosswalk.province));
+    const provinces = this.props.provinces.filter(province => vprommsProvinceKeys.indexOf(province.id.toString()) !== -1);
+    let accumulator = { field: this.props.fieldIds.length, total: Object.keys(this.props.VProMMs).length };
+    const provinceData = _.map(provinces, (province, key) => {
+      // very very crude answer to an issue of english name, will pluck when names are cleaned;
```

Comment:
If all you're doing here is fixing up the English display name of that one area, then you can delete this special handling. We should treat the display names in their spreadsheet as canonical, since we can't be expected to catch all issues.